### ISSUE TITLE
what does withSolid meaning?

### ISSUE THREAD

**Issue by Valar103769** [solidjs/solid#1237]
### Describe the bug

excuse me  I cannot understand the withSolid from the readme, can you expand talk? 

### Your Example Website or App

https://wc-todo.firebaseapp.com/

### Steps to Reproduce the Bug or Issue

none

### Expected behavior

none

### Platform

none

**Comment by ryansolid** [comment]
Yeah.  Solid Element uses my library [component-register](https://github.com/ryansolid/component-register). Component register has a mixin mechanism to add functionality. So `withSolid` is one of those that adds Solid as a renderer. We use that internally mostly, but I've exposed it incase people wanted to construct more involved behavior. If you are very much in Solid sphere of things this is probably not necessary but since the underlying library works that way, its why it is authored as such.
